project/src/main/webapp/index.html

```diff
@@ -23,18 +23,18 @@ <h1> FoodMood - your food order helper!</h1>
     <div id="query-form" class="query-form">
       <form id="cuisines-form" class="form-element-toolbar">
         <p>Select cuisines:</p>
-        <input type="checkbox" id="cuisine-italian" name="cuisine" value="italian" checked>
```

Comment:
why would we not want all these to be checked? I think it's easier for the user to keep all of them checked as default

Reply:
I think it is not comftable for the user (I actually changed it after running our app a few times and found it very annoying to delete all checks except one or two every time).
I changed the title so maybe it is more clear to the user now that if he doesn't want to limit the search then he doesn't need to pick anything. 
Anyway this doesn't have to do with this PR so it's my bad that this is here. Lets discuss UX next week for final touchups.

Reply:
we should discuss that. does that match the program? if they choose nothing they get everything? from my memory if they choose nothing they get an error

Reply:
Yes I removed the error that forces to check a cuisine in another PR that I was supposed to push and it got postponed. I added it now so we have it all here. Other then that the fetcher has no problem getting 0 prefered cuisines, it returns all sort of cuisines in this case (as we discussed) so it works fine, and IMO better.